extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php

```diff
@@ -1,75 +1,84 @@
 <?php
 
-class AnalyticsProviderQuantServe implements iAnalyticsProvider {
-
+class AnalyticsProviderQuantServe implements iAnalyticsProvider
+{
 	private $account = 'p-8bG6eLqkH6Avk';
 
-	function getSetupHtml($params=array()){
+	static function getQuantcastLabels() {
+		$keyValues = F::app()->wg->DartCustomKeyValues;
+		$quantcastLabels = array();
+		$labels = array(
+			'gnre' => 'Genre',
+			'media' => 'Media',
+			'theme' => 'Theme',
+			'tv' => 'TV'
+		);
+
+		if (gettype($keyValues) === 'string') {
+			foreach (explode(';', $keyValues) as $keyValue) {
+				$keyValue = explode('=', $keyValue);
```

Comment:
```suggestion
				list($key, $value) = explode('=', $keyValue);
```
Then `$key`/`$value` will be `string|null`

Reply:
Oh yeah, I forgot about that! Changed!